goss-testing/tests/ncn/goss-check-system-management-tools.yaml

```diff
@@ -0,0 +1,52 @@
+command:
+  assigned_ip-status:
+    exit-status: 0
+    exec: "ip addr show bond0.cmn0|grep -i inet"
+  default_gw:
+    exit-status: 0
+    exec: "ip route | grep default"
+    stdout:
+      - "default"
+  gw-reacheable:
+    exit-status: 0
+    exec: "ping -c 3 10.102.5.1"
+    stdout:
+      - "0% packet"
+    timeout: 30000
+  powerdns-status:
+    exit-status: 0
+    exec: "kubectl -n services get svc cray-dns-powerdns-cmn-udp"
+  switch_ping-status:
+    exit-status: 0
+    exec: "canu test --csm 1.2 --json --username admin --password ${SW_ADMIN_PASSWORD} |grep ping"
+    stdout:
+      - "PASS"
+    timeout: 60000
+  bgp_established_mellanox:
+    exit-status: 0
+    exec: "sshpass -p ${SW_ADMIN_PASSWORD} ssh admin@sw-spine-001.hmn cli \"enable\" \"show ip bgp vrf all summary\""
```

Comment:
sshpass is not installed on the NCNs currently. Is this being added?

Reply:
Yes its being handled by a bash script that calls this goss